### DISCUSSION TITLE
pixelWidth/Height

### DISCUSSION THREAD

**Discussion by adgad** [Financial-Times/content-tree#23] [Team Posts]
Spark are not publishing this, so Customer Products has to look it up from Origami (which is expensive)

https://github.com/Financial-Times/spark/issues/2559

**Comment by atanasdinov** [comment]
As mentioned in the issue itself, `pixelWidth` and `pixelHeight` are already supported in the image schema and can be published to and fetched from the C&M platform without any issues.
